src/static/css/cf-enhancements.less

```diff
@@ -968,7 +968,7 @@ textarea.input__long {
   name: Hide only when JS is active
   family: cf-core
   patterns:
-    - name: .u-reset
```

Comment:
`.u-reset` is the block [before this one](https://github.com/cfpb/cfgov-refresh/blob/cleanup/src/static/css/cf-enhancements.less#L947-L965). `.js-is-hidden` is [this block](https://github.com/cfpb/cfgov-refresh/blob/cleanup/src/static/css/cf-enhancements.less#L967-L986).